### ISSUE TITLE
Top bar is unlocatable in the chat view

### ISSUE THREAD

**Issue by churik** [status-im/status-react#16413]
# Problem
The top bar is missing in XML-structure of chat view, therefore there is no option to work with it from an automation perspective

![Appium 2023-06-27 15-48-21](https://github.com/status-im/status-mobile/assets/4557972/ba09b365-8eb8-40c3-956c-19cc96bd6e3b)

## Acceptance Criteria
1) can locate the top bar in the XML
2) accessibility-ids: on top-bar, on chat options

**Comment by flexsurfer** [comment]
hey @churik i think its in the tree but in another place it's related to animation implementation by @alwx might be he could help you to find it in the tree

**Comment by alwx** [comment]
It's definitely in the tree, it's impossible for something that is displayed to not be there :) but I guess the accessibility ids are missing and also the positioning makes it harder to find it visually.

**Comment by churik** [comment]
let's put accessibilty-ids on elements and I'll try to find it.

It looks to me the same as it was with the composer in the initial PR about top bar @alwx (it was overlapped by something and inaccessible)

**Comment by churik** [comment]
@alwx please, take a look - it makes it impossible to find anything; for me it looks like overlapped layer which I can't access even by text.
the same is for `Pinned messages` here
![photo_2023-07-24 10 39 33](https://github.com/status-im/status-mobile/assets/4557972/8b7e98b6-802b-46ba-b591-65bc3d8857b0)

**Comment by churik** [comment]
Vide for context:

https://github.com/status-im/status-mobile/assets/4557972/e1854381-1504-4c21-bae4-0cb8e07db586

The top bar doesn't exist on the tree, which makes impossible to automate anything relate to it, including pinned messages

**Comment by flexsurfer** [comment]
so the reason in this code
```
(def navigation-view
  {:z-index 1})
```

@alwx @ilmotta @Parveshdhull any ideas ?

**Comment by cammellos** [comment]
@alwx did you manage to figure this out?

**Comment by churik** [comment]
don't use it in automation, found a way to workaround
